docs/quickstart/introduction.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: "Introduction"
+description: "Nym connect quickstart"
+hide_title: false
+---
+
+# Introduction
+
+Nym Connect is a one-button interface that is easy to use. With Nym connect, you get 100% privacy on applications you use everyday. Presently, Nym connect has support for telegram, keybase and electrum wallets. This means that you can run sock5 proxy on these applications with Nym connect. More services coming soon.
+
+**Nym connect is still in experimental stage. If buggy please send us a [feedback report](https://dyno.gg/form/bbec84eb)**
```

Comment:
change 'if buggy' to something more formal e.g. 'if you experience problems or bugs, please send a feedback from'